Cargo.toml

```diff
@@ -17,6 +17,7 @@ name = "esplora_client"
 path = "src/lib.rs"
 
 [dependencies]
+async-std = "1.13.0"
```

Comment:
It should be an optional dependency, and only available when using the `async` feature.

Also, I don't think we need all the features in its default features, and only the `std` feature, is that right?

Reply:
I tried with only `std` feature and it seems that the `task` module is only available with "default"

Reply:
Yes, looks like it's behind the default feature [here](https://docs.rs/async-std/latest/src/async_std/task/mod.rs.html#133-164) using their [`cfg_default!`](https://github.com/async-rs/async-std/blob/fa29b508e138ee0c91873039bc28cb71a4546460/src/utils.rs#L231-L238) macro.